06-building-a-program-start-to-finish/solving-the-problem/Output/inverted-index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Inverted Index class
+ */
+class InvertedIndex {
+/**
+ * class constructor
+ */
+  constructor() {
+    this.indices = {};
+    this.documentCount = 0;
+  }
+/**
+ * normalizedText takes a string with numbers,
+ * spaces and symbols and returns a normalized string
+ * @param{String} text - The name of the string to be normalized
+ * @return{String} normalized string
+ */
+  normalizedText(text) {
+    this.text = text;
+    return this.text.toLowerCase().match(/[A-Za-z]+/g,
+      matched => matched.sort());
+  }
+/**
+ * @param{String} words - String to be filtered
+ * @return{Array} token - Array without duplicate words
+ */
+  uniqueWords(words) {
+    this.words = words;
+    const tokens = this.normalizedText(this.words);
+    return tokens.filter((item, index) =>
+                tokens.indexOf(item) === index);
+  }
+  /**
+   * Validate File
+   * @param {Object} file the selected file
+   * @returns {String} validation message
```

Comment:
JSDoc says you're returning a string but you're returning `check` which is a boolean.